src/webui/api/apicontroller.h

```diff
@@ -57,10 +58,12 @@ class APIController : public ApplicationComponent<QObject>
     void setResult(const QString &result);
     void setResult(const QJsonArray &result);
     void setResult(const QJsonObject &result);
-    void setResult(const QByteArray &result);
+    void setResult(const QByteArray &result, const QString &filename = {}, const QString &mimeType = {});
```

Comment:
> it would be better to swap parameter positions of `filename` and `mimeType`. `mimeType` is more general than `filename`, i.e. some endpoint can change the `mimeType` without using `filename` field.

Changed.